docker/Dockerfile

```diff
@@ -9,16 +9,29 @@ LABEL \
 COPY qemu-* /usr/bin/
 
 # System dependencies
-RUN apk add --no-cache tzdata nmap openzwave-dev openzwave-libs openzwave ffmpeg sqlite openssl gzip
+RUN apk add --no-cache tzdata nmap ffmpeg sqlite openssl gzip
 
+WORKDIR /tmp
+
+# Install OZW in a dedicated RUN to benefit from Docker cache on local dev'
+RUN apk add --no-cache --virtual .build-deps make gcc g++ python libffi-dev linux-headers git udev coreutils \
```

Comment:
Be careful, here the build deps are going to be included in the final image! 

Why? Because all docker layers are cached. 

So here, the `build-deps` are installed, and removed on a separate RUN statement, meaning that they won't be removed from the previous layer. 

Please roll back to the previous behavior :) You can maybe do a RUN statement dedicated to the installation of open-wave